docs/rich-text/converting-html.mdx

```diff
@@ -32,7 +32,7 @@ export const MyComponent = ({ data }: { data: SerializedEditorState }) => {
 }
 ```
 
-### Converting Lexical Blocks
+#### Lexical Blocks
```

Comment:
Hm not sure if this should be scoped under "On-demand (Recommended)", as this is valid for HTML field conversion as well